src/main/java/com/dsec/backend/controller/JobController.java

```diff
@@ -2,9 +2,13 @@
 
 import java.util.List;
 
+import com.dsec.backend.service.JobService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.security.oauth2.jwt.Jwt;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
```

Comment:
There is an unused import here. Please don't ignore the warnings that you get from your IDE.